### PR TITLE
Update: Totals can now have hidden items using isHidden

### PR DESCRIPTION
**Pull request by camjc**
- Hidden items are included the sum but not rendered.
- They should be displayed elsewhere for transparency to the user.
screen shot 2016-02-04 at 9 38 01 am

### CODE REVIEW

**Comment by antklim** [comment]
LGTM :tada:

File: src/components/alexandria/TotalsComponent.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ import React, { PropTypes } from 'react';
 
 const TotalsComponent = ({ toSum, valueFormatter }) => (
   <Grid>
-    {toSum.map(({ label, value }, index) =>
+    {_(toSum).filter(({ isHidden }) => !isHidden).map(({ label, value }, index) =>
```
Comment (omgaz) [inline src/components/alexandria/TotalsComponent.js]: omit isHidden

**Comment by omgaz** [comment]
LGTM

**Comment by omgaz** [comment]
Merging

**Pull request merged**